actor/v7action/service_instance_details.go

```diff
@@ -215,6 +220,19 @@ func (actor Actor) getServiceInstanceUpgradeStatus(serviceInstanceDetails Servic
 	}
 }
 
+func (actor Actor) getServiceInstanceBoundApps(serviceInstanceGUID string, omitApps bool) (boundApps []resources.ServiceCredentialBinding, warnings ccv3.Warnings, err error) {
+	switch omitApps {
```

Comment:
I find this a bit weird. I would find out if we need the apps before calling this method at all. It would made the intent clearer. From reading the caller code I thought maybe we were just fetching ids and the flag was to include the whole resource, but it turns out it is meant to skip the method altogether.